xbmc/games/addons/GameClient.cpp

```diff
@@ -276,6 +277,9 @@ bool CGameClient::OpenFile(const CFileItem& file, IGameAudioCallback* audio, IGa
   if (!InitializeGameplay(file.GetPath(), audio, video))
     return false;
 
+  m_inGameSaves.reset(new CGameClientInGameSaves(this, &m_struct));
```

Comment:
Move this to the constructor. I was just bit by a class that got the order of closing the resource wrong, so it's best to enforce the condition that it's always allocated.